magnolia-voting-app/src/main/java/com/example/votingapp/PollStatusValueProvider.java

```diff
@@ -0,0 +1,39 @@
+package com.example.votingapp;
+
+import java.util.Calendar;
+
+import javax.jcr.*;
+
+import com.vaadin.data.ValueProvider;
+
+/**
+ * PollStatusColumnDefinition
+ */
+public class PollStatusValueProvider implements ValueProvider<Node, String> {
+    
```

Comment:
Try to use logger in org.slf4j
private static final Logger log = LoggerFactory.getLogger(PollStatusValueProvider.class);